gridded_cayley_permutations/point_placements.py

```diff
@@ -308,4 +391,5 @@ def multiplex_map(self, cell: tuple[int, int]) -> MultiplexMap:
         return PartialMultiplexMap(cell, self.tiling.dimensions)
 
     def new_dimensions(self):
-        return (self.tiling.dimensions[0] + 2, self.tiling.dimensions[1])
+        """The new dimensions of the tiling after placement."""
+        return (self.tiling.dimensions[0] + 2, self.tiling.dimensions[1] + 2)
```

Comment:
This is a mistake, but perhaps partial point placement class no longer works now anyway with this inheritance